package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
What does this line do?

Reply:
From the interwebs: 

**_It stores an exact, versioned dependency tree rather than using starred versioning like package.json itself (e.g. 1.0.*). This means you can guarantee the dependencies for other developers or prod releases, etc. It also has a mechanism to lock the tree but generally will regenerate if package.json changes._**

I believe it just guarantees that npm install downloads the correct versions of node module dependancies.